test/e2e/storage/vsphere_scale.go

```diff
@@ -77,27 +74,22 @@ var _ = SIGDescribe("vcp at scale [Feature:vsphere] ", func() {
 		nodeVolumeMapChan = make(chan map[string][]string)
 
 		// Read the environment variables
-		volumeCountStr := os.Getenv("VCP_SCALE_VOLUME_COUNT")
-		Expect(volumeCountStr).NotTo(BeEmpty(), "ENV VCP_SCALE_VOLUME_COUNT is not set")
-		volumeCount, err = strconv.Atoi(volumeCountStr)
+		Expect(VCPScaleVolumeCount).NotTo(BeEmpty(), "ENV VCP_SCALE_VOLUME_COUNT is not set")
```

Comment:
I think VCPScaleVolumeCount, VCPScaleVolumesPerPod, VCPScaleInstances, SPBMPolicyName and StorageClassDatastoreName is not defined. 
Can you compile the code changes and make sure?